homeassistant/components/discovery/__init__.py

```diff
@@ -91,6 +90,7 @@
     SERVICE_HEOS,
     "harmony",
     "homekit",
+    SERVICE_IGD,
```

Comment:
You can remove it from here. It will now be discovered by the ssdp integration and should no longer be triggered via the `discovery` integration.

Reply:
Removed it.